code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -0,0 +1,84 @@
+	Widely known for their voracious nature and violent tendencies when left unfed for long periods of time. \
+	Most, if not all chimeras possess the ability to undergo some type of regeneration process, at the cost of energy."
+
+
+	//primitive_form = "Farwa"
+
+	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED //Whitelisted as restricted is broken.
+	flags = NO_SCAN //Dying as a chimera is, quite literally, a death sentence. Well, if it wasn't for their revive, that is.
+	appearance_flags = HAS_HAIR_COLOR | HAS_LIPS | HAS_UNDERWEAR | HAS_SKIN_COLOR | HAS_EYE_COLOR
+
+	flesh_color = "#AFA59E"
+	base_color 	= "#333333"
+	blood_color = "#14AD8B"
+
+/datum/species/chimera/handle_environment_special(var/mob/living/carbon/human/H) //Actually handles hunger.
+
+	if(H.stat) // If they're dead or unconcious they won't think about being all feral.
+		return
+	if(H.absorbed == 1) //If they get nomphed and absorbed.
+		return
+
+	if(H.nutrition > 50 && H.species.has_fine_manipulation == 0) //If they went feral then ate something.
+		H.species.has_fine_manipulation = 1
+		H << "<span class='primary'>You feel as if you're no longer feral, the feeling of intense hunger having now passed. You're sated. For the time being, at least.</span>"
+		return
+
+	else if(H.nutrition >= 300)
+		return
+
+	else if(H.nutrition < 300 && H.nutrition > 200)
+		if(prob(0.5)) //A bit of an issue, not too bad.
+			H << "<span class='info'>You feel extremely hungry. It might be a good idea to find some some food...</span>"
+			return
+
+	else if(H.nutrition <= 200 && H.nutrition > 100)
+		if(prob(0.5)) //Getting closer, should probably eat some food about now...
+			H << "<span class='warning'>You feel like you're going to starve and give into your hunger soon... It might would be for the best to find some [pick("food","prey")] to eat...</span>"
+			return
+
+	else if(H.nutrition <= 100 && H.nutrition > 50)
+		if(prob(1)) //Getting real close here! Eat something!
+			H << "<span class='danger'> You feel a sharp jabbing pain in your abdomen. You feel as if you're beginning to become feral, with food being the only thing on your mind. </span>"
+			return
+
+	else if(H.nutrition <= 50 && H.species.has_fine_manipulation == 1) //Should've eaten sooner!
+		H << "<span class='danger'><big>You suddenly feel a sharp stabbing pain in your stomach. You feel as if you've became completely feral, food the only thing on your mind.</big></span>"
+		H.emote("twitch")
+		H.species.has_fine_manipulation = 0 //Unable  to use objects.
+
+	for(var/mob/living/M in viewers(H))
+		if(M != H)
+			if(prob(0.5))//1 in 200 chance to tell them that person looks like food. This is so irregular so it doesn't pop up 24/7 during ERP.
+				H << "<span class='danger'> You feel a stabbing pain in your gut, causing you to twitch in pain.. It would be extremely wise to find some type of food... In fact, [M] looks extremely appetizing...</span>"
+				H.emote("twitch")
```

Comment:
This comes up a bit when you're dying, though. Like you're in crit, and it's like "You can't emote right now!"